Master-Controller/master-controller.py

```diff
@@ -284,9 +284,15 @@ def create_thing_visor_on_docker(tv_img_name, debug_mode, tv_id, tv_params, tv_d
 
         exposed_ports = {}
         if not debug_mode:
+            runtime_impl = 'runc'
+            isTty = True
+            if tv_img_name == 'thehajime/helloworld-tv-runu:1.0-docker':
+                runtime_impl = 'runu'
+                isTty = False
+
             # new entry: start thingVisor container and insert in system database
-            docker_client.containers.run(tv_img_name, name=tv_id, tty=True, detach=True, remove=True,
-                                         environment=env, publish_all_ports=True)
+            docker_client.containers.run(tv_img_name, name=tv_id, tty=isTty, detach=True, remove=False,
+                                         environment=env, publish_all_ports=True, runtime=runtime_impl)
```

Comment:
Hi Hajime,
Are those lines meant to be just for testing? Because we think are just too specific for the master-controller to handle.

Reply:
Thanks for the comment @ludfun.

I agree with you and I will drop those lines if there is no work around.

This block is intended to switch OCI runtime from the default one (runc) to others (e.g., runu, you can do whatever you wish).  Additionally, runu doesn't support tty option (`docker run -t`) thus drops as well.

I thought it can be implemented to add a command line argument to f4i.py and deliver the option to master-controller.py, but I also thought it's too much specific just wanting to replace OCI runtime.

OTOH, on k8s environment we can include those specific information into yaml files (https://github.com/fed4iot/VirIoT/pull/15/files#diff-21a47acd36921fad950ec7d9bee2aa53R15) so, I think it's okay.

Since our development may go forward k8s centric, I'm fine to drop docker support for runu/ukontainer and only upstream k8s part in this PR.

Reply:
I dropped docker CI.  Let me know how you think.